packages/common/src/docopts.ts

```diff
@@ -33,6 +36,20 @@ interface BasicLoadOpts {
   syncTimeoutSeconds?: number
 }
 
+/**
+ * Options that are used internally but aren't designed to be set by end users.
+ */
+export interface InternalOpts {
+  /**
+   * If true, when loading a stream log will throw an exception if any commit fails to apply.
+   * If false, the error will be logged, but the log application operation will return as much
+   * of the log as it was able to successfully apply.
+   * This option is used internally but is not designed to be set by user applications.
+   * @private
+   */
+  throwOnApplyCommitError?: boolean
```

Comment:
I'm not thrilled with putting this here where it's potentially exposed to users, but the only alternative I could come up with is to add a new parameter to `CeramicAPI.applyCommit()`, which will change both the core and HTTP client api, and still be visible to users anyway just in a different place.

The best long term solution, but one that would be a much bigger refactor, would be to have an `OperationContext` object threaded through just about every function in the code base.  We could use such an object to attach additional information and options related to the current top-level operation that we are executing.  That would let us distinguish in low level code whether we're doing something on behalf of a message that was received via pubsub, via an HTTP request, via a direct call into the core API, etc.